code/modules/mob/living/simple_animal/revenant/revenant.dm

```diff
@@ -78,6 +81,77 @@
 	if(world.time <= next_move)
 		return
 	A.attack_ghost(src)
+	if(ishuman(A) && in_range(src, A))
+		Harvest(A, src)
+
+
+/mob/living/simple_animal/revenant/proc/Harvest(mob/living/carbon/human/tar, mob/living/simple_animal/revenant/user = usr)
```

Comment:
Remove the user argument of this.
it's a revenant proc, you are NOT adding a revenant argument to it because that's stupid.